determined-ai/determined
# fix: modal theme color

## Description
[WEB-956]




## Test Plan
- Check if modal color looks good in both light/dark mode
- Check if `Trial Comparison`  and other modals looks good,




## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket






[WEB-956]: https://determinedai.atlassian.net/browse/WEB-956?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
